util/logging.c

```diff
@@ -91,14 +97,17 @@ int nvme_submit_passthru(int fd, unsigned long ioctl_cmd,
 {
 	struct timeval start;
 	struct timeval end;
-	int err;
+	int err = 0;
 
-	if (log_level >= LOG_DEBUG)
+	if (log_level >= LOG_DEBUG || dry_run)
```

Comment:
I'd rather not mix the log level with the `--dry-run` options. I suggest to update the documentation instead. 

Or if you don't agree, the `--dry-run` should then really only print the command nothing else.

Reply:
Yes right so fixed to output only command input data only without the result and err outputs for the `--dry-run` option.